### ISSUE TITLE
Skip links

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/client-web#5044]
## Description

When using a screen reader or keyboard control, it takes forever to get through the recent contributions block, because you cant skip from one contribution to the other or from one block to the other. The same holds for the navigation, every time you come across a tab menu, each item is mentioned. Skip links help to skip over these items so people go directly to the next section instead.

More about skip links: https://www.boia.org/blog/what-is-a-skip-link-and-how-does-it-work
https://css-tricks.com/how-to-create-a-skip-to-content-link/

## Acceptance criteria
Add skip links to the following sets. The links should become visible when they are focused upon, since not all everyone who uses keyboard navigation also has a screenreader that reads invisible content. Basically, it comes down to adding a skip link after the title of each block, allowing users to jump from one block immediately to the next one

### Design
**Design doesn't have to match completely, as long as the message is clearly visible. Also, the skip link may overlap with other elements, that is fine since it is only visible when focused upon**
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3VnQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5621d6140171875a56386ffe19bca57a273cc377/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMytYQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--234d65770412bb9b7a5cd78dea96fc96b78f1df3/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd2FZQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5ddfa4e9fa753451ef7a7534e6d2d5c10da18724/image.png)

https://www.figma.com/file/0S200A2Y99mB1DJfpJLb26/Skip-links?type=design&node-id=0%3A1&mode=design&t=0MkAxLZQvp02vLDM-1

### Main Navigation
- [ ] In front of first breadcrumb (white block with bold primary letters): "Skip main menu"

### Main navigation mobile
- [ ] Possibility to exit the menu

### Footer
- [ ] In front of first item (Terms?): "Skip footer"

### Homepage
- [ ] In each block (13), add a skip link after the title, stating: "Skip to next block"

### Innovation library
- [ ] In each block (2), add a skip link after the title, stating: "Skip to next block"

### Forum
- none

### Challenges
- [ ] In each block (3), add a skip link after the title, stating: "Skip to next block"

### Contributors
- [ ] In each block (2), add a skip link after the title, stating: "Skip to next block" 

### User/org profile
- [ ] In each block (3), add a skip link after the title, stating: "Skip to next block" 

### Innovation pack profile
- [ ] In each block (4), add a skip link after the title, stating: "Skip to next block" 

### Callouts
- [ ] Add a skip link after the title, stating: "Skip to next block" 

#### Dashboard
- [ ] In each block (5), add a skip link after the title, stating: "Skip to next block" 
#### Community
- [ ] In each block (3), add a skip link after the title, stating: "Skip to next block" 
#### Challenges
- [ ] In each block (3), add a skip link after the title, stating: "Skip to next block" 
#### Knowledge base
- [ ] In each block (2), add a skip link after the title, stating: "Skip to next block" 

#### Dashboard
- [ ] In each block (5), add a skip link after the title, stating: "Skip to next block" 
#### Contribute
- [ ] In each block (2), add a skip link after the title, stating: "Skip to next block" 
#### Opportunities
- [ ] In each block (3), add a skip link after the title, stating: "Skip to next block" 

#### Dashboard
- [ ] In each block (5), add a skip link after the title, stating: "Skip to next block" 
#### Contribute
- [ ] In each block (2), add a skip link after the title, stating: "Skip to next block" 
#### About
- [ ] In each block (11), add a skip link after the title, stating: "Skip to next block" 

## Additional Context

To make this work, all blocks need to be identifiable, they need anchors. This is difficult with callouts.
There is a quick way and a proper way, we choose the quick way :) this will work for most of the time.
